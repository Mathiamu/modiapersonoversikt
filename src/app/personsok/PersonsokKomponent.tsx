import * as React from 'react';
import PersonsokSkjema from './PersonsokSkjema';
import PersonsokResultat from './PersonsokResultat';

function PersonsokKomponent() {
    return (
        <div>
            <PersonsokSkjema />
            <PersonsokResultat />
        </div>
    );
}

export default PersonsokKomponent;
