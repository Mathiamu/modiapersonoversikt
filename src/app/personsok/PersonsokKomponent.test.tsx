import * as React from 'react';
import * as renderer from 'react-test-renderer';
import TestProvider from '../../test/Testprovider';
import PersonsokKomponent from './PersonsokKomponent';

test('viser personsÃ¸k-komponent', () => {
    const personsokKomponent = renderer.create(
        <TestProvider>
            <PersonsokKomponent />
        </TestProvider>
    );

    expect(personsokKomponent.toJSON()).toMatchSnapshot();
});
