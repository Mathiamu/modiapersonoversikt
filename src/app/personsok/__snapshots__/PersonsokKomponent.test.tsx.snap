// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`viser persons√∏k-komponent 1`] = `null`;
